src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -1012,7 +1073,7 @@ namespace OpenMS
         s_completiontime_ = xercesc::XMLString::transcode("completionTime");
         s_precision_ = xercesc::XMLString::transcode("precision");
         s_byteorder_ = xercesc::XMLString::transcode("byteOrder");
-        s_pairorder_ = xercesc::XMLString::transcode("pairOrder");
+        s_contentType_ = xercesc::XMLString::transcode("contentType");
```

Comment:
our test input-files are still 2.x, we are writing (and reading) 3.x during the tests. No problem there.